scipy/spatial/transform/_rotation.pyx

```diff
@@ -15,12 +15,20 @@ np.import_array()
 
 # utilities for empty array initialization
 cdef inline double[:] _empty1(int n) noexcept:
+    if n == 0:
+        return array(shape=(1,), itemsize=sizeof(double), format=b"d")[:0]
     return array(shape=(n,), itemsize=sizeof(double), format=b"d")
 cdef inline double[:, :] _empty2(int n1, int n2) noexcept :
+    if n1 == 0:
+        return array( shape=(1, n2), itemsize=sizeof(double), format=b"d")[:0]
```

Comment:
Here still hanging " " .

Reply:
Fixed in 7d097440d4bf6703c357f8a34d7be862cd34d943